messaging/src/main/java/org/axonframework/eventhandling/ReplayToken.java

```diff
@@ -296,6 +349,14 @@ public boolean covers(TrackingToken other) {
         return currentToken != null && currentToken.covers(other);
     }
 
+    @Override
```

Comment:
Same nit as on the other tokens - shouldn't we reuse the method logic in `covers` for those that are identical.